includes/class-wc-product-accommodation-booking.php

```diff
@@ -410,6 +410,17 @@ public static function get_check_times( $type, $product_id = 0 ) {
 		public function get_duration( $context = 'view' ) {
 			return 1;
 		}
+
+		/**
+		 * Get the duration time interval in minutes.
+		 *
+		 * Duration unit is always one night.
+		 *
+		 * @return int duration in minutes.
+		 */
+		public function get_time_interval_in_minutes() {
+			return 1440; // 60 minutes * 24 hours
```

Comment:
To match what the parent class does, we should look to use WordPress constants here:

```suggestion
			return DAY_IN_SECONDS / MINUTE_IN_SECONDS;
```